index.html

```diff
@@ -137,9 +137,10 @@ <h1 id="title">*PROPOSED* Web Performance Working Group Charter</h1>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
-        <p>Web developers are building sophisticated applications where application performance is a critical feature. Web developers need the ability to observe the performance characteristics of their applications, and they want the ability to write more efficient applications, using well-defined interoperable methods.</p>
+        <p>Web developers are building sophisticated applications where application performance is a critical feature. Web developers want the ability to observe the performance characteristics of their applications, and they want the ability to write more efficient applications, using well-defined interoperable methods.
+        Their methods must be both secure and privacy-enabling by design, based on the current Web browser security model.</p>
```

Comment:
In principle, I agree with the last sentence, but there's some nuance here I'm not convinced about.. 

- What specifically constitutes a "secure" performance API? Scoped to secure origins only, something else?
- Ditto for "privacy preserving". Some API's _may_ expose new bits and that tradeoff is made on a feature by feature basis, based on detailed analysis of the tradeoffs. 

If the intent is to highlight that expose APIs / information must respect web browser security model (which I agree with), I'd propose we reword this as.. 

> "... using well-defined interoperable methods that conform to the current Web browser security model."

WDYT?

Reply:
@igrigorik I like your modification. The point is to highlight the tradeoffs for careful evaluation.

Reply:
👍 

I assume you'll make the text update? ty ahead of time!

Reply:
Yes, done.